tests/functional/avatar.js

```diff
@@ -94,36 +94,6 @@ define([
         .end();
     },
 
-    'go to settings with an email NOT selected to see change link should not see one': function () {
```

Comment:
Since this is no longer possible, the [BackMixin](https://github.com/mozilla/fxa-content-server/pull/2748/files#diff-71782557c742ba10d42de19c28d3a7d1R86) should be removed from the view.